xls/modules/fpsqrt_32.x

```diff
@@ -0,0 +1,232 @@
+// Copyright 2021 The XLS Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+// This code is based on the go double sqrt implementation,
+// available at https://golang.org/src/math/sqrt.go
+// This implementation is in turn based on 
+// FreeBSD's /usr/src/lib/msun/src/e_sqrt.c
+// The lisence for both sources along with comment from
```

Comment:
s/liscense/license

Reply:
I don't think I follow - what's "s" here?

Reply:
Oh, sorry - vim command for find-and-replace - in other words, replace "liscense" with "license". :)